# Configuration file for the microphone
tts:
  default_param:
    region_name: "us-west-2"
    language: "it-IT"
    outdir: "$(find harmoni_tts)/temp_data"
    wav_header_length: 24
    voice: "Bianca"
